recipes/glog/all/conandata.yml

```diff
@@ -11,4 +11,3 @@ sources:
 patches:
   "0.5.0":
     - patch_file: "patches/0001-fix-msvc-snprintf.patch"
-      base_path: "source_subfolder"
```

Comment:
Could please add `patch_description` and `patch_type` as described here: https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches

Reply:
@SpaceIm could you do that? I dont know why this this required. Thanks!

Reply:
@the-nic You are supposed to do that. Please, take a look on my suggestion.

Reply:
I didnt write that patch and have no access to msvc. I also just wanted to fix the libunwind things. All the migrations a a bit out of scope for the actual issue, TBH. I can of course apply the suggestion